src/Scope.ts

```diff
@@ -850,6 +862,52 @@ export class Scope {
         });
         return results;
     }
+
+    public getAllClassMemberCompletions() {
+        let results = new Map<string, CompletionItem>();
+        let filesSearched = new Set<BscFile>();
+        for (const file of this.getAllFiles()) {
+            if (isXmlFile(file) || filesSearched.has(file)) {
+                continue;
+            }
+            filesSearched.add(file);
+
+            const statementHandler = (s: ClassMethodStatement) => {
+                if (!results.has(s.name.text) && s.name.text.toLowerCase() !== 'new') {
+                    results.set(s.name.text, {
+                        label: s.name.text,
+                        kind: CompletionItemKind.Method
+                    });
+                }
+            };
+            const fieldStatementHandler = (s: ClassFieldStatement) => {
+                if (!results.has(s.name.text)) {
+                    results.set(s.name.text, {
+                        label: s.name.text,
+                        kind: CompletionItemKind.Field
+                    });
+                }
+            };
+            file.parser.ast.walk(createVisitor({
```

Comment:
Considering you have all the classes references, it would be better to iterate over the statements of the class object directly instead of using a walker which will walk everything in depth.